server.py

```diff
@@ -22,17 +22,16 @@
 
 
 gpt_neo_key = os.environ.get("GPTNEO", None)
-mongo_uri = os.environ.get("MONGO", "mongodb+srv://admin:admin@reportai.ks0reyi.mongodb.net/")
 headers = {"Authorization": gpt_neo_key}
-
-client = MongoClient(mongo_uri, server_api=ServerApi("1"))
+MONGO_URI = "mongodb+srv://admin:admin@reportai.ks0reyi.mongodb.net/"
+client = MongoClient(MONGO_URI, server_api=ServerApi("1"))

```

Comment:
I think using os.environ.get() will be better than hard-coding,
This is how os.environ.get() works, it tries to fetch MONGO key from env variable, if nothing is defined in env variable takes in the next argument as default which is `mongodb+srv://admin:admin@reportai.ks0reyi.mongodb.net/`, 

Did you face an issue while using `os.environ.get` ?

Reply:
So do u mean to say, if the user uses a different mongodb cluster they should specify the mongo_uri in their ``.env``` file ? 

Because if they just store the key, it will be a problem.

Reply:
```python3
MONGO_URI = os.environ.get(mongo, <uri>)
client = MongoClient(MONGO_URI, server_api = ServerApi("1"))
```
For this to work, mongo that is fetched from the ```.env``` file must be some other mongo-uri and not a key.